src/ngraph/runtime/cpu/cpu_builder.cpp

```diff
@@ -103,6 +103,12 @@
 #include "ngraph/type/element_type.hpp"
 #include "ngraph/util.hpp"
 
+#include "ngraph/runtime/cpu/mkldnn_invoke.hpp"
+#include "ngraph/runtime/cpu/mkldnn_utils.hpp"
+#include "ngraph/runtime/cpu/op/convert_layout.hpp"
```

Comment:
nit: `convert_layout.hpp` is included in line 100. so this one could be deleted.

Reply:
thanks for catching it. I will fix it.